docs/mpcnc/intro.md

```diff
@@ -0,0 +1,69 @@
+
+- All axis can be any length you prefer, anything over 3 1/2′ (1M) would require small mid-span supports to increase rigidity, of course smaller is better.  The kit comes with enough belt for up to 48″ of total outer X and Y axis dimensions (eg 24″x24″, 36″x12″ or any other combination).
+
+- The smaller you make this the faster you can move it and the more rigid/accurate it will be, and more importantly the easier it will be to get the desired accuracy.
+
+- The linear motion is ball bearings on electrical conduit (or stainless steel)… seriously, easy to use and source. More information on this [here](https://www.v1engineering.com/assembly/machine-size/). If this machine isn’t big enough I have also designed the [LowRider CNC](https://www.v1engineering.com/lowrider-cnc/), all the same specs and resolution just designed to be bigger!
+
+ 
+## Power
+
+- X and Y axis are powered by 2 stepper motors each, and a single stepper for the Z axis. The standard would be NEMA 17 in any torque preferably 42 OZ/in and above (the kit come with 76 OZ/in). No need for NEMA 23’s or their required larger drivers,  torque is not one of this machines issues.
+
+- Belts are used for there accuracy and price. Small belts are used because each axis has two of them. Ballscrews are expensive, require tuning, and periodic maintenance / adjustments.
+
+ 
+## Control
+
+- This is all controlled by any control board you like. The UltiMachine boards are recommended for the improved design and robust safety features. You can still use the common Ramps 1.4 or any others boards as well. There are other [Marlin](https://github.com/MarlinFirmware/Marlin) based boards, [GRBL](https://github.com/grbl/grbl), and regular CNC boards as well.
+
+ 
+## Software
+
+- There are plenty of free or really inexpensive software options available for 3D printing, CAM, and CAD. I suggest [Fusion 360](http://www.autodesk.com/products/fusion-360/overview), [Repetier-host](http://www.repetier.com/), [ESTLCam](http://www.estlcam.com/), ect.
+
+ 
+## DIY
+
+- Besides using either common imperial or metric hardware, the rest of the machine is easily 3D printable, RepRap style! Full list of hardware [here](https://www.v1engineering.com/blog/parts/), and the files for the printed parts can be had [here](https://www.thingiverse.com/thing:724999).
+
+- All of these things can be easily assembled with basic hand tools, no specialty tools, power tools, or precision cuts required.
+
+![!Simple MPCNC](https://www.v1engineering.com/wp-content/uploads/2015/07/IMG_20150802_16352001.jpg){: width="450"}
+![!Fancy MPCNC](https://www.v1engineering.com/wp-content/uploads/2018/04/IMG_20180409_184626.jpg){: width="450"}
+[Build Thread](https://www.v1engineering.com/forum/topic/red-black-and-wheels/)
+
+## More info
+
+To see some of the capabilities, and some interesting builds check out some [videos](https://www.v1engineering.com/videos/), or the project [gallery](https://www.v1engineering.com/videos/gallery/).
```

Comment:
The gallery is a really good selling point. I would like to see it higher up, and maybe in one of those fancy boxes so people see it. Better yet, pick a few cool images, and make a section, like:

# Check out the gallery

The [gallery] shows some of the accomplishments by MPCNCs over the last 5 years.

Cool pic

Cool pic

Cool pic